environments/india/postgresql.yml

```
@@ -53,9 +53,10 @@ dbs:
         shards: [820, 1023]
         pgbouncer_host: pgbouncer0
   custom:
-    - django_alias: icds-ucr-citus
-      name: icds-ucr-citus
-      django_migrate: True  # ICDS dashboard models are stored here
+    # remove citus from django
+    # django_alias: icds-ucr-citus
+    # django_migrate: True  # ICDS dashboard models are stored here
```

```
BadValueError: Property django_migrate is required.
```
looks like it needs to to change this to
```suggestion
    django_migrate: False
```